### DISCUSSION TITLE
Update installer and TCa quadrant

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4068] [General]
Hi simmers,

after the last update of the fbw installer the folder has got a new name in the community/packages folder. So far so good. The other folder to adjust the paramerers for the detents on the quadrants in the offical folder is not available?! The detents are now not correct. How or where to find the file to change the parameters to get the detents working again? I am using the custom fbw version.

Greets Klaus

**Comment by CptBayer** [comment]
Sorry folks...

I got it... went to the wrong folder... Localstate is it ..


Greets Klaus

**Reply by 9Konrad9** [reply]
Hi Klaus,

i have the same problem. I found the config. file and there were the same values as before. So i did reinstall the custom FBW and reset the quadrant, but the detents are still not working. Maybe you can explain it to me? Thanks. 

Greets Konrad

**Reply by CptBayer** [reply]
Hi Konrad,

I use the custom FBW mode. It is for me the only version where this tweek is working. After installing the custom FBW you will find in the localstate folder a folder called work:

Microsoft.FlightSimulator_8wekyb3d8bbwe\LocalState\packages\flybywire-aircraft-a320-neo\work

Here you change the values as followed:

[Throttle]
Log = true
Enabled = true
ReverseOnAxis = true
ReverseIdle = true
DetentDeadZone = 2.0
DetentReverseFull = -1.00
DetentReverseIdle = -0.70
DetentIdle = -0.44
DetentClimb = -0.10
DetentFlexMct = 0.53
DetentTakeOffGoAround = 1.00

That will work fine for me. Detents behaviour is back to normal after changing the values. 

Greets klaus

**Reply by frogmonkey2** [reply]
This appears to be a tweak from the original Github for the Thrustmaster in the FBW NEO?... did you alter the value sets?

**Reply by frogmonkey2** [reply]
[Throttle]
Log = true
Enabled = true
ReverseOnAxis = true
ReverseIdle = true
DetentDeadZone = 2.5
DetentReverseIdle = -0.90
DetentReverseFull = -1.00
DetentIdle = -0.42
DetentClimb = 0.06
DetentFlexMct = 0.53
DetentTakeOffGoAround = 1.00

**Reply by frogmonkey2** [reply]
this my configuration from this site

**Reply by frogmonkey2** [reply]
> 
> 
> this my configuration from this site

https://github.com/flybywiresim/a32nx/tree/fbw/docs

**Reply by CptBayer** [reply]
I saw different values on different pages. This values above worked for me ...

greets klaus

**Reply by frogmonkey2** [reply]
Well TBH I can try each. The one I show works fine. However it's off just slightly in the climb detent, nothing to get upset over... as long as I can fly! The throttle quad and reversers are awesome. Just got email about my side mounts being available...😁

**Reply by frogmonkey2** [reply]
FYI I tried each yours and the original-(mine)- the original/mine works far better with the indents! you need to try the one made by the author here. https://github.com/flybywiresim/a32nx/tree/fbw/docs

**Reply by xxeonn** [reply]
[TLDR] - Fix for Missing INI

1. Install A32nx Mod using installer.
2. Open MS Flight Simulator
3. Load A32nx in a Flight
4. Use FlyPad and click on Gear icon to open settings.
5. Click Calibrate to calibrate throttle.
6. If you check back in work folder the ThrottleConfiguration.ini should now be there.

**Comment by B1900Mech** [comment]
This doesn't seem to work for me,I have the required entry.

**Reply by B1900Mech** [reply]
Disregard wrong folder.

**Comment by Mickepilot** [comment]
About a week ago GitHub had and update and I finally found the work/throttle config file at a complete new place. So I was glad that I got the Thrustmaster TCA Throttle Quadrant working correct again. (after adjusting the values)
Yesterday evening, there was several new updates/bug fixes.... via MS Store  ... and within the app itself... and also GitHubs A32NX. I used the "Installer" for the "Custom FBW" installation. Since then the work/throttle config file is gone. Even after a flight, no config file (for adjusting the throttles values) is being created.
Anybody with similar situation? Any help appreciated.

**Reply by frogmonkey2** [reply]
uninstall or delete the community file mod. and then go to the work files again and recheck if there are no conflicting files left to delete and try a new install?

**Reply by frogmonkey2** [reply]
create one?

**Reply by mojoair** [reply]
> About a week ago GitHub had and update and I finally found the work/throttle config file at a complete new place. So I was glad that I got the Thrustmaster TCA Throttle Quadrant working correct again. (after adjusting the values)
> Yesterday evening, there was several new updates/bug fixes.... via MS Store ... and within the app itself... and also GitHubs A32NX. I used the "Installer" for the "Custom FBW" installation. Since then the work/throttle config file is gone. Even after a flight, no config file (for adjusting the throttles values) is being created.
> Anybody with similar situation? Any help appreciated.

Same issue here. No matter what I do, since the latest updated, It will not auto generate the Throttle config file in the work folder.

**Comment by frogmonkey2** [comment]
yes, somethings changed since their latest upgrade? I opened MSFS and the
splash screen readouts changed and now recognizes Fly_By_Wire
... I re-installed it as it asked me... MSFS has a new update... the 1st
install went automatically to 'Stable' my settings for throttle only work
of the experimental... now they don't?... I've had nothing but trouble with
the Thrustmaster quad and the FBY mod since day one... only a few flights
with the parameters correct, very frustrating to say the least...😞

On Fri, Mar 26, 2021 at 4:48 PM Mickepilot ***@***.***> wrote:

> About a week ago GitHub had and update and I finally found the
> work/throttle config file at a complete new place. So I was glad that I got
> the Thrustmaster TCA Throttle Quadrant working correct again. (after
> adjusting the values)
> Yesterday evening, there was several new updates/bug fixes.... via MS
> Store ... and within the app itself... and also GitHubs A32NX. I used the
> "Installer" for the "Custom FBW" installation. Since then the work/throttle
> config file is gone. Even after a flight, no config file (for adjusting the
> throttles values) is being created.
> Anybody with similar situation? Any help appreciated.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/flybywiresim/a32nx/discussions/4068#discussioncomment-536479>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASZPF3DKTDYO437R7T67B2TTFUMODANCNFSM4ZSUKQRQ>
> .
>


-- 
Donald

**Reply by Mickepilot** [reply]
Wow... well, I'm happy again, I just found the **Throttle Quadrant calibration settings** ..... in the "FlyPad"!!!
(after installing all MS and Asobo updates yesterday, and also the "Custom FBW" via the "FlyByWire" Installer)
The detents can now actually be even more precise adjusted, separate for the left and right engine throttle!
Be sure to set both "Reverser On Axis" and "Dual Axis" to "On".
Be sure also to calibrate the Thrustmaster TCA Quadrant before entering the detent values. For me these values works nice:
LEFT (Axis 1): Reverse: Start=-1 End=-0.98, Rev.Idle: Start=-0.7 End=-0.65, Idle: Start=-0.56 End=-0.45, CLB: Start=0.2 End=0.3, FLX: Start=0.7 End=0.8, TO/GA: Start=0.95 End=1
RIGHT (Axis 2): Reverse: Start=-1 End=0.98, Rev.Idle: Start=-0.7 End=-0.65, Idle: Start=-0.55 End=-0.44, CLB: Start=0.2 End=0.3, FLX: Start=0.7 End=0.8, TO/GA: Start=0.97 End=1
I still can not activate the different auto brake functions with the Thrustmasters Add-On switched selector (position nr. 22 - 26)
Hopefully some at GitHub will get it working.

**Reply by frogmonkey2** [reply]
all this just for a throttle?... my goodness... thrustmeter's configuration will NOT change as per neo throttles go... so why make it so hard not to provide a 'default' profile for us?

**Reply by Mickepilot** [reply]
Ohhh, sorry folks, I just realized that my setting above is not valid for the standard Thrustmaster Detent plastic part (There are two of them mounted on the underside of the throttle quadrant, each secured with two screws)
Instead, the above values is for the following easy mod:
https://www.shapeways.com/product/ZRUW892G8/realistic-detents-for-thrustmaster-tca-quadrant
optionId=201217014&li=marketplace
With these parts the Thrustmaster throttle handles has a more realistic range between Idle and CLB

**Comment by frogmonkey2** [comment]
I inserted your values and got some results... however this is a BIG
problem that really should NOT be one in my opinion... Why is it so hard to
set up the #1 throttle for Airbus in the world? the value date sets BEFORE
the new flyPad work well enough for me... now I can't get it past
climb/flex without clutches... FBW needs to have a fix... not so hard on
their end if they 'test' with  throttle quad like 'us' 'we' that fly the mod

On Fri, Mar 26, 2021 at 8:01 PM Mickepilot ***@***.***> wrote:

> Wow... well, I'm happy again, I just found the *Throttle Quadrant
> calibration settings* ..... in the "FlyPad"!!!
> The detents can now actually be even more precise adjusted, separate for
> the left and right engine throttle!
> Be sure to set both "Reverser On Axis" and "Dual Axis" to "On".
> Be sure also to calibrate the Thrustmaster TCA Quadrant before entering
> the detent values. For me these values works good:
> LEFT (Axis 1): Reverse: Start=-1 End=-0.98, Rev.Idle: Start=-0.7
> End=-0.65, Idle: Start=-0.56 End=-0.45, CLB: Start=0.2 End=0.3, FLX:
> Start=0.7 End=0.8, TO/GA: Start=0.95 End=1
> RIGHT (Axis 2): Reverse: Start=-1 End=0.98, Rev.Idle: Start=-0.7
> End=-0.65, Idle: Start=-0.55 End=-0.44, CLB: Start=0.2 End=0.3, FLX:
> Start=0.7 End=0.8, TO/GA: Start=0.97 End=1
> I still can not activate the different auto brake functions with the
> Thrustmasters Add-On switched selector (position nr. 22 - 26)
> Hopefully some at GitHub will get it working.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/flybywiresim/a32nx/discussions/4068#discussioncomment-536715>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASZPF3GIRJPRIOQROPSGOSDTFVDAPANCNFSM4ZSUKQRQ>
> .
>


-- 
Donald

**Reply by tracernz** [reply]
We have tested, and experienced no issues. You need to understand that the “experimental” version is just what’s written on the tin, and there is no end-user documentation or anything because it is not yet ready for that audience.

**Reply by frogmonkey2** [reply]
Well, it took quite a bit of searching and even having Andreas Guther answer me directly to help. He even suggested his DATA values for his Thrustmaster.

**Reply by frogmonkey2** [reply]
> 
> 
> Well, it took quite a bit of searching and even having Andreas Guther answer me directly to help. He even suggested his DATA values for his Thrustmaster.

Andreas Guther
Mine looks like this (you cannot use it 1:1 because I use a modded version of the TCA):
[THROTTLE_COMMON]
REVERSE_ON_AXIS = true
[THROTTLE_AXIS_1]
REVERSE_LOW = -1.00
REVERSE_HIGH = -0.98
REVERSE_IDLE_LOW = -0.80
REVERSE_IDLE_HIGH = -0.60
IDLE_LOW = -0.47
IDLE_HIGH = -0.41
CLIMB_LOW = 0.25
CLIMB_HIGH = 0.35
FLEX_MCT_LOW = 0.70
FLEX_MCT_HIGH = 0.90
TOGA_LOW = 0.98
TOGA_HIGH = 1.00
[THROTTLE_AXIS_2]
REVERSE_LOW = -1.00
REVERSE_HIGH = -0.98
REVERSE_IDLE_LOW = -0.80
REVERSE_IDLE_HIGH = -0.60
IDLE_LOW = -0.47
IDLE_HIGH = -0.41
CLIMB_LOW = 0.25
CLIMB_HIGH = 0.35
FLEX_MCT_LOW = 0.70
FLEX_MCT_HIGH = 0.90
TOGA_LOW = 0.98
TOGA_HIGH = 1.00

**Comment by frogmonkey2** [comment]
what do you mean here ( Be sure also to calibrate the Thrustmaster TCA
Quadrant before entering the detent values)
TIA I'm having zero luck getting going, you have me close

On Fri, Mar 26, 2021 at 8:01 PM Mickepilot ***@***.***> wrote:

> Wow... well, I'm happy again, I just found the *Throttle Quadrant
> calibration settings* ..... in the "FlyPad"!!!
> The detents can now actually be even more precise adjusted, separate for
> the left and right engine throttle!
> Be sure to set both "Reverser On Axis" and "Dual Axis" to "On".
> Be sure also to calibrate the Thrustmaster TCA Quadrant before entering
> the detent values. For me these values works good:
> LEFT (Axis 1): Reverse: Start=-1 End=-0.98, Rev.Idle: Start=-0.7
> End=-0.65, Idle: Start=-0.56 End=-0.45, CLB: Start=0.2 End=0.3, FLX:
> Start=0.7 End=0.8, TO/GA: Start=0.95 End=1
> RIGHT (Axis 2): Reverse: Start=-1 End=0.98, Rev.Idle: Start=-0.7
> End=-0.65, Idle: Start=-0.55 End=-0.44, CLB: Start=0.2 End=0.3, FLX:
> Start=0.7 End=0.8, TO/GA: Start=0.97 End=1
> I still can not activate the different auto brake functions with the
> Thrustmasters Add-On switched selector (position nr. 22 - 26)
> Hopefully some at GitHub will get it working.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/flybywiresim/a32nx/discussions/4068#discussioncomment-536715>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASZPF3GIRJPRIOQROPSGOSDTFVDAPANCNFSM4ZSUKQRQ>
> .
>


-- 
Donald

**Comment by frogmonkey2** [comment]
glad your flying again... I hope to join you one day...

On Fri, Mar 26, 2021 at 8:01 PM Mickepilot ***@***.***> wrote:

> Wow... well, I'm happy again, I just found the *Throttle Quadrant
> calibration settings* ..... in the "FlyPad"!!!
> The detents can now actually be even more precise adjusted, separate for
> the left and right engine throttle!
> Be sure to set both "Reverser On Axis" and "Dual Axis" to "On".
> Be sure also to calibrate the Thrustmaster TCA Quadrant before entering
> the detent values. For me these values works good:
> LEFT (Axis 1): Reverse: Start=-1 End=-0.98, Rev.Idle: Start=-0.7
> End=-0.65, Idle: Start=-0.56 End=-0.45, CLB: Start=0.2 End=0.3, FLX:
> Start=0.7 End=0.8, TO/GA: Start=0.95 End=1
> RIGHT (Axis 2): Reverse: Start=-1 End=0.98, Rev.Idle: Start=-0.7
> End=-0.65, Idle: Start=-0.55 End=-0.44, CLB: Start=0.2 End=0.3, FLX:
> Start=0.7 End=0.8, TO/GA: Start=0.97 End=1
> I still can not activate the different auto brake functions with the
> Thrustmasters Add-On switched selector (position nr. 22 - 26)
> Hopefully some at GitHub will get it working.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/flybywiresim/a32nx/discussions/4068#discussioncomment-536715>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASZPF3GIRJPRIOQROPSGOSDTFVDAPANCNFSM4ZSUKQRQ>
> .
>


-- 
Donald

**Comment by P1NBA11ER** [comment]
I'm trying to use the FlyPad to calibrate my TCA but it will not correctly align reverse thrust. Am I missing something? I've got everything else working going through the FlyPad settings but anything beyond idle sits at idle only. I had this working previously using the ThrottleConfiguration.ini file method but that no longer seems to work with the FlyPad option available.

**Reply by Paytheon** [reply]
P1NBA11ER, watch this video on how to calibrate the TCA throttle. https://www.youtube.com/watch?v=OCCJOQyWyq8&t=822s

Hope it helps.

**Reply by P1NBA11ER** [reply]
Thanks for posting, I follow the channel but missed this video somehow. Adding a min/max value seems to have fixed this for me. It was jumping all over otherwise.

**Reply by P1NBA11ER** [reply]
Well, it kinda worked. The throttle at the climb detent (visibly in the cockpit and mechanically) was not in climb according to the AP and the plane was overspeeding. Unfortunately, the design of the throttle has it sit quite precariously in one single position between the idle and climb detent, and the the slightest movement causes it to pop into one or the other, so basically useless. This is with the custom FBW so I assume some of this may be a problem in this version of the A32NX. It did work for me fine though, prior to having to do this reconfiguration through the FlyPad.
